### PR TITLE
Can view joined tables with big-spenders, mallrats, sassy cats, and c…

### PR DESCRIPTION
**Pull request by SharjeelSIqbal**
…ustomers who enjoy that sort of thing 
Screen Shot 2021-08-18 at 10 49 00 AM (2)

### CODE REVIEW

**Comment by SharjeelSIqbal** [comment]
Screen Shot 2021-08-18 at 10 59 44 AM (2)
Screen Shot 2021-08-18 at 10 41 08 AM (2)
Screen Shot 2021-08-18 at 11 26 10 AM (2)
Screen Shot 2021-08-18 at 11 54 22 AM (2)

File: sql-join/magic-mallrats.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+select "films"."title",
+"customers"."firstName",
+"customers"."lastName",
+"rentals"."customerId",
+"rentals"."rentalId",
+"inventory"."inventoryId"
```
Comment (thebearingedge) [inline sql-join/magic-mallrats.sql]: double check the instructions for each SQL file. Only specific columns were requested.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Just some minor cleanup

File: sql-join/where-are-they.sql [hunk 2]
```diff
@@ -0,0 +1,15 @@
+select "customers"."firstName",
+"customers"."lastName",
+"payments"."amount",
+"payments"."customerId",
```
Comment (thebearingedge) [inline sql-join/where-are-they.sql]: We don't need any customer info for this one, just full addresses.

image

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: sql-join/where-are-they.sql [hunk 3]
```diff
@@ -0,0 +1,11 @@
+select "addresses"."line1",
+"addresses"."district",
+"cities"."name" as "city",
+"countries"."name" as "country"
+from "customers"
+join "payments" using ("customerId")
```
Comment (thebearingedge) [inline sql-join/where-are-they.sql]: ???

File: sql-join/where-are-they.sql [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+join "addresses" using ("addressId")
+join "cities" using ("cityId")
+join "countries" using ("countryId")
+order by "payments"."amount" desc
+limit 10;
```
Comment (thebearingedge) [inline sql-join/where-are-they.sql]: ??

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**